index.fr.md

```diff
@@ -0,0 +1,226 @@
+
+## Tirez des conclusions et créez un rapport {#drawing}
+
+Faites attention si vous tirez des conclusions d'évaluations ou d'études limitées. Les résultats obtenus avec deux personnes handicapées ne peuvent être généralisés et appliqués à toutes les personnes dans la même situation de handicap ou présentant d'autres handicaps. Retrouvez plus d'informations dans la section [Attention](#caution) ci-dessous.
+
+Incluez dans votre rapport le cadre de votre étude et les paramètres de l'évaluation, tels que les méthodes de test et les caractéristiques des utilisateurs. Par exemple, si une étude a inclu uniquement des personnes aveugles, le rapport devra préciser que celle-ci n'a pas évalué la conformité aux standards d'accessibilité et qu'elle ne s'applique pas à toutes les personnes handicapées. Bien que les études de petite envergure peuvent souvent fournir des informations utiles, elles ne sont pas assez solides pour fournir une signification statistique.
+
+## Note à l'attention des ergonomes {#ut-access}
+
+Lors de la recherche d'obstacles à l'accessibilité spécifiques, le protocole est généralement différent d'un test d'utilisation général. Par exemple :
+
+-   vous utiliseriez certainement un système de réfléxion à voix haute et une forte interaction avec l'organisateur des tests
+-   la collecte des données serait axée sur la compréhension des erreurs relatives aux obstacles à l'accessibilité, plutôt que sur le temps passé à réaliser une tâche ou sur la satifaction utilisateur
+-   les tâches seraient axées sur des parties spécifiques qui pourraient potentiellement poser problème, plutôt que sur l'utilisation générale du site
+
+Notez qu'il est également important d'évaluer d'autres facteurs pour les utilisateurs handicapés. Ces facteurs comprennent : l'utilisation générale, la satisfaction utilisateur, et d'autres critères semblables.
+
+La section Plus d'informations ci-dessous inclut une aide supplémentaire à l'attention des ergonomes.
+
+## Plus d'informations et aide {#fmi}
+
+Ce document aborde brièvement quelques points d'un sujet très complexe. De nombreuses ressources traitant des autres aspects liés à l'inclusion des utilisateurs dans l'évaluation sont disponibles sur le Web.
+
+-  **[[Inclure des utilisateurs dans les projets Web pour une accessibilité plus optimale et plus facile]](/planning/involving-users/) constitue un préambule à ce document**. Cette page traite de questions plus vastes sur l'inclusion des utilisateurs dès le début de la conception d'un site Web, du développement des outils, des standards, et des autres projets Web.
+-   [Demandez : Intégrer l'accessibilité tout au long de la conception](http://www.uiaccess.com/accessucd/overview.html) fournit de l'aide sur l'intégration de l'accessibilité tout au long de la conception de sites Web et d'autres produits. Le chapitre [Test d'utilisation pour évaluer l'accessibilité](http://www.uiaccess.com/accessucd/ut.html) comprend les différentes phases suivantes :
```

Comment:
Many times here and below : better to use "test d'ergonomie" than "test d'utilisation"